.github/actions/notify-build-status/README.md

```diff
@@ -79,9 +79,9 @@ jobs:
       - id: check
         uses: elastic/apm-pipeline-library/.github/actions/check-dependent-jobs@current
         with: ${{ toJSON(needs) }}
-      - run: ${{ steps.check.outputs.isSuccess }}
       - uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
         with:
+          status: "${{ steps.check.outputs.isSuccess ? 'success' : 'failure' }}"
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
```

Comment:
inputs section has not been updated with this new input parameter

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2130/commits/68b5c9d6b4040632bc76b90d051db297cc17d2c6